modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -96,14 +96,50 @@ The contents of the file resemble the following example:
 ----
 // An additional step for following this procedure when using oc-mirror as part of the disconnected install process.
 ifdef::oc-mirror[]
-. Save the file either as `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
+. Save the file as either `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`. 
+
+.. If the `.docker` or `$XDG_RUNTIME_DIR/containers` directories do not exist, create one by entering the following command:
++
+[source,terminal]
+----
+$ mkdir -p <directory_name>
+----
++
+Where `<directory_name>` is either `~/.docker` or `$XDG_RUNTIME_DIR/containers`.
+
+.. Copy the pull secret to either the `~/.docker` directory or the `$XDG_RUNTIME_DIR/containers` directory by entering the following command:
++
+[source,terminal]
+----
+$ cp <path>/<pull_secret_file_in_json>
+----
++
+Where `<path>/<pull_secret_file_in_json>` is either `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
```

Comment:
In ```Save the file either as ~/.docker/config.json or $XDG_RUNTIME_DIR/containers/auth.json``` description, seem <path>/<pull_secret_file_in_json> not ~/.docker/config.json or $XDG_RUNTIME_DIR/containers/auth.json

Reply:
Hi,

Sorry. I'm not sure I understand this feedback. I did update to what I think you are saying. Can you check again? 

Thanks